src/lambda_function.py

```diff
@@ -1,20 +1,19 @@
 import time
 
 from webdriver_wrapper import WebDriverWrapper
+from selenium.webdriver.common.keys import Keys
 
 
 def lambda_handler(*args, **kwargs):
     driver = WebDriverWrapper()
 
     driver.get_url('https://www.google.es/')
 
-    driver.set_input_value('//input[@id="lst-ib"]', '21 buttons')
-
-    driver.click('//center//img[@alt="Google"]')
```

Comment:
This line and the above are the only ones that need to change to make the example work :)